### PR TITLE
add step by step for billing

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2693 Document billing QFC](https://app.clickup.com/t/2192114/QF-2693)

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -22,6 +22,31 @@ Go to the [registration page](https://app.qfield.cloud/accounts/signup/), enter
...
+
+Adding billing information to the account is needed to use QFieldCloud Pro plan or Organization plan. 
+
+For a **Pro plan**, follow these steps :
+
+1. Click on the username up-right the page
+2. Click on "Upgrade to pro"
+3. In the "Subscription" section, click on "Upgrade"
+4. In Pro section, click on "Activate"
+5. Check the box "I agree with Terms of Service." and click on "Proceed"
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:35]:
```suggestion
5. Carefully read the "Terms of Service" and agree to the terms of service, then "Proceed"
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 2]
```diff
@@ -22,6 +22,33 @@ Go to the [registration page](https://app.qfield.cloud/accounts/signup/), enter
...
+
+For an **Organization plan**, follow these steps :
+
+1. Click on the username up-right the page
+2. Click on "Create organization"
+3. Click on "Create"
+4. Choose a name for your organization with only 150 characters or fewer, letters, digits, and @/./+/-/_
+5. Click on "Create"
+6. Choose how much storage is needed
+7. Check the box "I agree with the Terms of Service." and click on "Proceed"
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:46]: @lucienicolier can please also update here?

Comment (lucienicolier) [reply documentation/get-started/tutorials/get-started-qfc.en.md]: @m-kuhn done

**Comment by lucienicolier** [comment]
@suricactus can you take a look, please? :)

**Review by suricactus — APPROVED** [review]
Thanks! Sorry for the late review!

**Pull request merged**